testing/trino-tests/src/test/java/io/trino/execution/TestConnectorEventListener.java

```diff
@@ -31,7 +32,9 @@
 @Test(singleThreaded = true)
 public class TestConnectorEventListener
 {
-    private final EventsCollector generatedEvents = new EventsCollector();
+    private final EventsCollector generatedEvents = new EventsCollector(EventFilters.builder()
+            .setSplitCompletedFilter(event -> false)
```

Comment:
Also this is not testing for split completed, so explicitly ignore them incase we somehow ended up with an extra split, which is very unlikely given this is a VALUES query